src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs

```diff
@@ -51,6 +51,9 @@ protected Command()
         [Option(typeof(NuGetCommand), "Option_ConfigFile")]
         public string ConfigFile { get; set; }
 
+        [Option(typeof(NuGetCommand), "Option_ForceInvariant")]
+        public bool ForceInvariant { get; set; }
```

Comment:
Would it be more clear if this was ``-English`` or ``-EnglishText``? The name here is very correct, I just wonder if users will read it as something that affects the install or lock file.